core/builder.go

```diff
@@ -27,6 +27,10 @@ type BuildCfg struct {
 	// If online is set, the node will have networking enabled
 	Online bool
 
+	// If permament then node should run more expensive processes
+	// that will improve performance in long run
+	Permament bool
```

Comment:
this is not a good name. the node wont ever be `Permanent`. you're using it here to signify "NotEphemeral", so just use `Ephemeral` and set it to false.